pkg/ui/react-app/src/pages/graph/PanelList.tsx

```diff
@@ -139,6 +143,10 @@ const PanelList: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix = ''
   const { response: storesRes, error: storesErr, isLoading: storesLoading } = useFetch<StoreListProps>(
     `${pathPrefix}/api/v1/stores`
   );
+  const { response: flagsRes, error: flagsErr, isLoading: flagsLoading } = useFetch<FlagMap>(
+    `${pathPrefix}/api/v1/status/flags`
+  );
+  const defaultStep = flagsRes.data && 'query.default-step' in flagsRes.data ? flagsRes.data['query.default-step'] : '1s';
```

Comment:
```suggestion
  const defaultStep = flagsRes?.data?.['query.default-step'] || '1s';
```

This can be simplified by using Optional Chaining. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Much cleaner 😃